src/app/modules/group/pages/group-edit/group-edit.component.html

```diff
@@ -0,0 +1,9 @@
+<ng-container *ngIf="state$ | async as state">
+
+  <p-progressSpinner *ngIf="state.tag === 'fetching'"></p-progressSpinner>
+
+  <p *ngIf="state.tag === 'error'">Error while loading the group</p>
+
+  <p>Group loaded successfully!</p>
```

Comment:
you do not to check the state to print successfully

Reply:
fixed